svc-bie-kafka/src/main/java/gov/va/vro/services/bie/model/BieMessagePayload.java

```diff
@@ -0,0 +1,23 @@
+package gov.va.vro.services.bie.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.extern.jackson.Jacksonized;
+
+@JsonInclude
+@Setter
+@Getter
+@Builder
+@Jacksonized
```

Comment:
Do you need `@JsonInclude` or `@Jacksonized`?

Reply:
Didn't need it! Removed.